packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -1208,253 +1199,6 @@ new PrismaClient({
       })
     }
 
-    private _bootstrapClient() {
```

Comment:
Some new year resolutions.

Reply:
I see the call of this removed in line 446 as well - but where did the logic go?

Reply:
We now use JS custom constructor return values
https://github.com/prisma/prisma/pull/10764/files/568f6c1aa293564ba9bf54bd0689462b3ba449a4#diff-09784fc0113a735572cff48990efa35434fb6ec6ec8876845ec9cf423642e5fdR449